db/db.go

```diff
@@ -67,6 +69,7 @@ type Result struct {
 // Rows represents the outcome of an operation that returns query data.
 type Rows struct {
 	Columns []string        `json:"columns,omitempty"`
+	DeclTypes []string      `json:"decltypes,omitempty"`
```

Comment:
Let's pick a different name for this. `decltypes` is too low-level. How about just `type`?